test/system/books_test.rb

```diff
@@ -0,0 +1,20 @@
+require "application_system_test_case"
+
+class BooksTest < ApplicationSystemTestCase
+  attr_reader :book
+
+  def setup
+    @book = books(:ror_interview_bible)
+  end
+
+  test "visiting the index" do
```

Comment:
nitpick suggestion

Usually the test should describe the outcome and not the process.

```suggestion
  test "should display the title" do
```